pyproject.toml

```diff
@@ -59,7 +59,9 @@ mypy-boto3-secretsmanager = "^1.24.0"
 mypy-boto3-ssm = "^1.24.0"
 mypy-boto3-appconfig = "^1.24.0"
 mypy-boto3-dynamodb = "^1.24.0"
```

Comment:
Looking at https://pypi.org/project/mypy-boto3/, it says _"This tool is not longer required for annotations to work. Do not use it, install [boto3-stubs](https://pypi.org/project/boto3-stubs/) instead."_

Reply:
Did boto3-stubs work for you when creating interfaces and such? 

I had varying success, especially with mypy

Reply:
I didn't use either - saw this library here, and wanted to learn more about it, then saw this notice and commented back :)

Reply:
@heitorlessa I also used boto3-stubs in other projects. Do you have any specific example that it didn't work for you and mypy-boto3 does?